_posts/2024-12-23-cold-start-search.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title:  "Solving the cold start search problem in OpenSearch"
+authors:
+ - aswath
+ - allan
+ - kolchfa
+date: 2024-12-23
+categories:
+ - technical-posts
+meta_keywords:   
+meta_description: 
```

Comment:
Please update the meta with the following: 

_meta_keywords:_ cold start search, OpenSearch refresh interval, search latency, search performance optimization

_meta_description:_ Explore the cold start search problem in OpenSearch after upgrading from older Elasticsearch versions. Learn about the root causes and discover practical solutions to optimize search performance for various workload scenarios.

Reply:
Done